sapl/materia/models.py

```diff
@@ -413,14 +413,12 @@ class DocumentoAcessorio(models.Model):
                              on_delete=models.PROTECT,
                              verbose_name=_('Tipo'))
     nome = models.CharField(max_length=50, verbose_name=_('Nome'))
-    data = models.DateField(blank=True, null=True, verbose_name=_('Data'))
+    data = models.DateField(verbose_name=_('Data'))
```

Comment:
Este campo ser `NOT NULL` terá impacto negativo na migração, pois já existem muitos Documentos Acessórios **sem data**. Essa validação deve ser feita a nível de formulário, ou seja, no `forms.py`, deixando o atributo `required=True`.